ddcz/templates/public.html

```diff
@@ -1,210 +1,62 @@
-                                {% endif %}
-                                <p class="stats">
-                                    Hlasů: {{ user.profile.reputace_rozdel }} <br>
-                                    Reputace: {{ user.profile.reputace }}
-                                </p>
-                                <p class="admin_link"><a href="{{ user.profile.profile_url }}">Statistiky</a>
-                                {% if user.is_staff %}
-                                    <a href="{% url 'dragon:dashboard' %}">Správa</a>
-                                {% endif %}
-                                </p>
-                                <form method="post" action="{% url 'ddcz:logout-action' %}">
-                                    {% csrf_token %}
-                                    <input type="submit" value="Odhlaš">
-                                </form>
-                            {% else %}
-                                <p>Registrace</p>
-                                <form method="post" action="{% url 'ddcz:login-action' %}">
-                                    {% csrf_token %}
-                                    {{ login_form }}
-                                    <input type="submit" value="Přihlásit">
-                                </form>
-                                <p><a href="{% url 'ddcz:password-reset' %}">Zapomněl jsem heslo</a></p>
-                            {% endif %}
-                            </div>
-                        </li>
-                    </ul>
-                    <ul class="nav_border">
-                        <li><span class="list_heading">Doupě jinde</span>
-                            <ul class="list_box">
-                                <li><a href="https://discord.gg/U6TEhAKV">Discord</a></li>
-                                <li><a href="https://www.facebook.com/DraciDoupecz-47586881642/">Facebook</a></li>
-                            </ul>
-                        </li>
-                    </ul>
-                </nav>
-            </div>
-        </div>
-        <footer id="global-footer">
-            <p>© 2018-2021 Almad, design © 2002 a 2018-21 by Almad <a href="https://github.com/dracidoupe/graveyard/graphs/contributors">a ostatní přispěvatelé</a>. DrD, Dračí doupě a Altar jsou ochranné známky společnosti ALTAR. Všechna práva vyhrazena. Žádná část těchto stránek nesmí být reprodukována, publikována ani jinak využita bez svolení Redakce serveru DraciDoupe.cz nebo svolení autorů jednotlivých příspěvků.</p>
-            <p>Tato verze stránek je open source a <a href="https://github.com/dracidoupe/graveyard/">je možné přispět k jejímu vývoji</a> podle <a href="https://ddcz.readthedocs.io/en/latest/">dokumentace</a>.</p>
+
+        {% if skin == "historic" %}
+        {% include "base/historic/top_menu.html" %}
+        {% endif %}
+
+        {% with "base/"|add:skin_for_include|add:"/left_menu.html" as lmenu %}
```

Comment:
I think we should invent a better name, since it seems we now have two separate concepts:

* Skin as presented to the user
* "Underlying skin code" 

The second one needs a name, I think